### PR TITLE
Add main structure, books page, and categories page

### PR DESCRIPTION
**Pull request by ahmed-al-farouq**
What I did:
- Created the main structure
- Created navbar and added router
- Created books and categories page
- Created form in books page
- Added style

### CODE REVIEW

File: src/components/Books.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+import Book from './booksPage/Book';
+import Form from './booksPage/Form';
+
+const Hr = styled.hr`
+  width: 86rem;
+  border: solid 1px #e8e8e8;
+  margin: 2.5rem auto 1.813rem auto;
+`;
```
Comment (dasileker) [inline src/components/Books.jsx:10]: - [ ] Please follow the best CSS practice, you can create a CSS file and add all the styling in it instead.

File: src/components/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 55px;
+  border-radius: 5px;
+  color: #ffffffad;
+  font-family: sans-serif;
+  font-weight: 400;
+  font-size: 30px;
+`;
```
Comment (dasileker) [inline src/components/Categories.jsx:18]: - [ ] The same here also.

File: src/components/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: #0290ff;
+`;
+
+const Ul = styled.ul`
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  list-style: none;
```
Comment (dasileker) [inline src/components/Navbar.jsx:30]: - [ ] Also here

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @ahmed-al-farouq,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.jsx (hunk 1 above)
Comment (ahmed-al-farouq) [reply src/components/Books.jsx]: Hi @dasileker,
I made the required changes.
Thanks for the feedback.

**Review by AshakaE — APPROVED** [review]
### Hi @ahmed-al-farouq  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**